sub/osd.c

```diff
@@ -85,6 +87,7 @@ const struct m_sub_options osd_style_conf = {
         .margin_y = 22,
         .align_x = -1,
         .align_y = -1,
+        .fonts_dir = "~~/fonts",
```

Comment:
This does work and mpv will expand it internally, but I think we should avoid it. Unless the user actually sets the option, we should leave it null. Instead what you can do is just check if `fonts_dir` is null and if so use `fonts` when getting the path (like the current behavior). If not, then you can actually use this option.

Reply:
Done.